integr8ly/integreatly-operator
# Operator-sdk bump to 1.7.2

# Description
Operator-sdk bump to 1.7.2

# Updates
- finalizers renames to <component>.<org>/finalizer
- some dependencies updates
- minor change around make image/build docker command

# Verification
- need sanity check on the finalizers re-work (code)
- run the initial install from this branch (bump to sdk 1.7.2) on byoc cluster with cluster storage set to false
```
curl -Lso operator-sdk https://github.com/operator-framework/operator-sdk/releases/download/v1.7.2/operator-sdk_linux_amd64 && \
chmod +x operator-sdk && \
sudo mv operator-sdk /usr/local/bin
```
- verify that upgrade from master to this branch work as expected (via OLM), also on byoc cluster with cluster storage false
- verify uninstallation and confirm that everything deletes correctly (including AWS resources being correctly deleted)
